### ISSUE TITLE
[Bug Report]: Missing Charmed effect name and image

### ISSUE THREAD

**Issue by Cicopath** [chronosacaria/MCDungonesArtifacts#42]
### Describe the bug; What happened?

The Charmed effect is missing its name and image.
![image](https://github.com/user-attachments/assets/45030883-ad52-402e-b347-40baa8b8dbb0) (via JEED)

### Expected Behavior

The Charmed effect should be named "Charmed" and have an image associated with the effect.

### Mod Version

Fabric 1.20.1; mod version 4.0.3

### Minecraft Launcher

CurseForge

### Operating System

Windows

### Modpack

Creating my own

### Additional context

Seems that the Charmed effect name was never added to the language file: https://github.com/chronosacaria/MCDungeonsArtifacts/blob/1.20/src/main/resources/assets/mcdar/lang/en_us.json#L51. Also apparent for the other language files.

There also doesn't seem to be an image for the Charmed effect: https://github.com/chronosacaria/MCDungeonsArtifacts/blob/1.20/src/main/resources/assets/mcdar/textures/mob_effect/charmed.png.

**Comment by chronosacaria** [comment]
@Cicopath Yeah, that is fair. Since it is never player facing, we didn't make an icon or translate it, but in the interest of best practices, we really should. Thanks for opening an issue about this
